Cabal/src/Distribution/Simple/Errors.hs

```diff
@@ -11,9 +11,12 @@
 
 module Distribution.Simple.Errors
   ( CabalException (..)
+  , CabalInstallException (..)
```

Comment:
This PR should _not_ put anything into the `Cabal` source tree -- the additions should be made only to modules in `cabal-install`'s source tree. This lets us maintain modularity. Conceptually, `cabal-install` depends on `Cabal`, therefore the latter should not know anything about the former, only vice versa.